tests/core/interfaces/outbound/dataops/test_dataops.py

```diff
@@ -287,7 +288,7 @@ def test_getting_default_adapter_from_interface(self):
                     columns=[
                         ColumnValidation(
                             unique_name="col1",
-                            data_type="varchar",
```

Comment:
is `varchar` still supported by `dataguard`? If not, there will be other instances in the code to adapt.